docs/other-topics/migrations.md

```diff
@@ -445,57 +453,57 @@ The configuration file is by default a JSON file called `config.json`. But somet
 Thankfully, the Sequelize CLI can read from both `.json` and `.js` files. This can be setup with `.sequelizerc` file. You just have to provide the path to your `.js` file as the `config` option of your exported object:
 
 ```js
-const path = require('path');
+const path = require("path");
```

Comment:
Don't change the code style of the document (this is valid for all code style related changes that occurred)

Reply:
> Don't change the code style of the document (this is valid for all code style related changes that occurred)

I just turn off my prettier extension in my vscode, it seems still working for markdown :/ I will revert

Reply:
> Don't change the code style of the document (this is valid for all code style related changes that occurred)

My prettier extension got messed up, let me fix it